test/functional/update/update-search-file-index.bats

```diff
@@ -49,7 +49,7 @@ test_setup() {
 	assert_status_is 0
 	expected_output=$(cat <<-EOM
 		Update started
-		Preparing to update from 10 to 100
+		Preparing to update from 10 to 100 \(in format staging\)
```

Comment:
Why escaping parens in this case but not others?

Reply:
assert_regex_is_output vs assert_in_output as the testing function.